scipp-jupyter-demo.Dockerfile

```diff
@@ -42,6 +52,16 @@ RUN conda install --yes \
 # Avoid weird tornado AttributeError
 RUN pip install --upgrade nbconvert
 
+# Get code for SANS direct-beam iteration demo
+RUN OWD=$(pwd) && \
```

Comment:
```suggestion
RUN PWD=$(pwd) && \
```

Reply:
Doesn't `PWD` get overwritten when I change directory?

Reply:
Ah, yes. My point was more that it was a bad name.

Reply:
I figured that it was unnecessary anyway, removed now.